returnn/dataset.py

```diff
@@ -149,3 +149,110 @@ def run(self):
                 hdf_writer.insert_batch(numpy.asarray([[speaker_index]], dtype="int32"), [1], [segment_name])
 
         hdf_writer.close()
+
+
+class ExtractSeqLensJob(Job):
+    """
+    Extracts sequence lengths from a dataset for one specific key.
+
+    Also see: :class:`i6_experiments.users.schmitt.corpus.statistics.GetSeqLenFileJob`
```

Comment:
I am not sure I understand why this is referenced here, this job is a local user job which can be changed or removed at any point, and it works quite different.

Reply:
I had this in my own local copy. I just forgot to remove this.